tests/test_shell.py

```diff
@@ -49,6 +49,7 @@ def setUp(self, request):
             pythonpath = [os.getcwd()] + pythonpath.split(":")
         os.environ["PYTHONPATH"] = ":".join(pythonpath)
         self.tmpdir = TemporaryDirectory(prefix="signac_")
+        self.project_name = f"Project({repr(os.path.realpath(self.tmpdir.name))})"
```

Comment:
Is this attribute used? I don't see what this line is meant to do.

Reply:
Probably an unnecessary holdover from rebasing.

Reply:
I think it's a holdover from an old commit where the `__str__` was defined differently.